commands/commands.go

```diff
@@ -45,8 +45,12 @@ var (
 	cmdLine         string
 
 	// errors for common purposes
-	errNotLoggedIn      = "user not logged in"
-	errConflictingFlags = "conflicting flags used"
+	errNotLoggedIn           = "user not logged in"
+	errConflictingFlags      = "conflicting flags used"
+	errClusterIDNotSpecified = "cluster ID not specified"
+	errClusterNotFound       = "the cluster specified could not be found"
+	errInternalServerError   = "an internal server error occurred"
+	errNotAuthorized         = "not authorized"
```

Comment:
These should be typed errors. Handling errors with strings is an anti pattern in golang. We have types and type assertion. Lets use them.